v2/azure-arm.yaml

```diff
@@ -1588,6 +1588,25 @@ objectModelConfiguration:
 #        $exportAs: ServersDatabasesWorkloadGroupsWorkloadClassifier
 #        $supportedFrom: v2.0.0
   storage:
+    2022-09-01:
+      StorageAccounts_TableService:
+        $exportAs: StorageAccountsTableService
```

Comment:
Normally I am all in favor of using the latest API version, especially when adding a new resource or a totally new RP. In this case though we already have lots of this RPs peer items but at a lower API version and now we're going to have a mismatch in consistency.

2021-04-01 StorageAccounts_BlobService, but 2022-09-01 `StorageAccounts_FileService`.

I wonder if we should either add support for BlobService/FileService at the old version rather than this new verison, OR add support for this new version for the existing types so that one version has all of the types.

That would have the nice benefit of updating storage to support a newer API which does seem possibly desirable?

Reply:
In that case, I personally would prefer to add an older matching resource version for the new resources. That said, users running the older version of parent/peer resources who want to implement the new resource, would then still have to install the newer version which would not be consistent and would be the same scenario as we are in right now or they'll have to migrate the resources to be on the same API version to be consistent.

Reply:
> In that case, I personally would prefer to add an older matching resource version for the new resources. That said, users running the older version of parent/peer resources who want to implement the new resource, would then still have to install the new resource with a newer version which would not be consistent and would be the same scenario as we are in right now or they'll have to migrate the resources to be on the same API version to be consistent.

Not sure I follow this.

If you added for `2022-09-01` all of the existing StorageAccount stuff too, then we would have exactly the same experience for people who are using the old `StorageAccount` and want `TableService`, with the added benefit that we also support the new StorageAccount with its new features?